### PR TITLE
Bug 2041507: Fix issues in AddCapacity Modal

### PR DESCRIPTION
**Pull request by bipuladh**
StorageClassDropdown now references the component.
Screenshot from 2022-01-19 15-36-51

### CODE REVIEW

File: packages/odf/modals/add-capacity/add-capacity-modal.tsx [hunk 1]
```diff
@@ -57,6 +57,8 @@ import {
 } from '../../utils/ocs';
 import './add-capacity-modal.scss';
 
+const StorageClassDropdown = (ScModule as any).StorageClassDropdown;
```
Comment (afreen23) [inline packages/odf/modals/add-capacity/add-capacity-modal.tsx:60]: Why this ?

**Review by afreen23 — CHANGES_REQUESTED** [review]

**Comment by bipuladh** [comment]
Changes in the kubevirt SDK, Previously it was exporting the component now it is exporting the whole module but under the same name. 
Reference: 
https://github.com/openshift/console/commit/e94be3265f2ced9c4aca1587ee3ac011a665d534#diff-5e83ab3d6fc56a1651d0621bac9b3d901cadaa18c868b99c4582262fbbb1e453L175-L176

File: packages/odf/modals/add-capacity/add-capacity-modal.tsx (hunk 1 above)
Comment (bipuladh) [reply packages/odf/modals/add-capacity/add-capacity-modal.tsx:60]: See https://github.com/red-hat-storage/odf-console/pull/101#issuecomment-1016277766

**Review by afreen23 — APPROVED** [review]
/lgtm

File: packages/odf/modals/add-capacity/add-capacity-modal.tsx (hunk 1 above)
Comment (afreen23) [reply packages/odf/modals/add-capacity/add-capacity-modal.tsx:60]: Makes sense now, thanks.

**Pull request merged**